### DISCUSSION TITLE
Blinking of WS2812 LEDs

### DISCUSSION THREAD

**Discussion by HelmiBeh** [arendst/Tasmota#22447] [Q&A]
Is there a possibility to add a blinking feature to every single addressable LED as an extension to the LED<x> command? Or can this be done by a rule? Need this feature to blink several LEDs, while others are lighting with a static color.

**Comment by sfromis** [comment]
If you want *every single* led you can use the normal `Power blink` (along with `BlinkTime` and `Blinkcount`, but then you also post words like you did not want *every single* led after all.

With ESP8266, rules are the way to have local automations, including to issue `Led` commands for a segment of a strip. With ESP32, you can also do stuff with the Berry language.

**Reply by HelmiBeh** [reply]
Please can You give me an example for power blink? This has only a function for me, if I use any Scheme. If I use the LED command, it blinks with a previous scheme, but the single LED colors are gone. So I think, I have to program the blink function in Node-Red.

**Reply by sfromis** [reply]
The command `Power blink` (I *already* gave you the example) only works for the whole strip, including a scheme, but not relevant if you want to control just a part of the strip.

But indeed, if you want to blink only part of the strip, an automation is needed. You'd not "have to" use Node-Red for that, you can also use rules or with ESP32, also Berry.
